cgo/lapack.go

```diff
@@ -108,6 +111,51 @@ type Implementation struct{}
 
 var _ lapack.Float64 = Implementation{}
 
+// Dgeqp3 computes a QR factorization with column pivoting of the
+// m×n matrix A: A*P = Q*R using Level 3 BLAS.
+//
+// The matrix Q is represented as a product of elementary reflectors
+//  Q = H[0] H[1] . . . H[k-1], where k = min(m,n).
```

Comment:
I think we have settled on using `[ ]` for indexing of vectors and matrices, and `_` for subscripting (as here).